reliability/README.md

```diff
@@ -0,0 +1,92 @@
+# Reliability
```

Comment:
This is a good new section but I think some of it should at least tie back in to maybe the agile process possible.  e.g. pointing to best practices on retroing some of the mishaps out and using existing ceremonies to call those items out maybe

Reply:
Good idea! added a note to the bottom, let me know if you think there's a better way to include more info here